src/Microsoft.ML.Data/Training/TrainerEstimatorBase.cs

```diff
@@ -62,6 +63,20 @@ public TrainerEstimatorBase(IHost host, SchemaShape.Column feature, SchemaShape.
             WeightColumn = weight;
         }
 
+        /// <summary>
+        /// Normal metadata that we produce for score columns.
+        /// </summary>
+        protected static IEnumerable<SchemaShape.Column> MetadataForScoreColumn(bool isNormalized = false)
+        {
+            var cols = new List<SchemaShape.Column>();
+            cols.Add(new SchemaShape.Column(MetadataUtils.Kinds.ScoreColumnSetId, SchemaShape.Column.VectorKind.Scalar, NumberType.U4, true));
+            cols.Add(new SchemaShape.Column(MetadataUtils.Kinds.ScoreColumnKind, SchemaShape.Column.VectorKind.Scalar, TextType.Instance, false));
+            cols.Add(new SchemaShape.Column(MetadataUtils.Kinds.ScoreValueKind, SchemaShape.Column.VectorKind.Scalar, TextType.Instance, false));
+            if (isNormalized)
+                cols.Add(new SchemaShape.Column(MetadataUtils.Kinds.IsNormalized, SchemaShape.Column.VectorKind.Scalar, BoolType.Instance, false));
```

Comment:
On the Score column generated from the OVA tests, i see yet another column: "Slot Names"
Dees that need to be added to the list of metadata here?